templates/readme.md

```diff
@@ -0,0 +1,61 @@
+# DOT Graph template
+
+```
+crane template -t path/to/graph.tmpl -o output/myGraph.dot
+```
+
+Generate a [DOT](http://en.wikipedia.org/wiki/DOT_(graph_description_language)) file representing the dependency graph. Bold nodes represent the containers declared in the config (as opposed to non-bold ones that are referenced in the config, but not defined). Targeted containers are highlighted with color borders. Solid edges represent links, dashed edges volumesFrom, and dotted edges net=container relations.
+
+# Systemd template
```

Comment:
I understood this was given as an example/suggestion, but I just hate to see documentation examples which don't work, so running them as part of the automated testing would prevent that.